### PR TITLE
issue 272 make a direction for turing machine

### PR DESCRIPTION
**Pull request by eddie7788**

### CODE REVIEW

File: frontend/src/components/InputDialogs/InputDialogs.tsx [hunk 1]
```diff
@@ -389,14 +389,19 @@ const InputDialogs = () => {
             />
           </InputWrapper>
           <InputWrapper>
-            <Input
+            <select
```
Comment (ire4ever1190) [inline frontend/src/components/InputDialogs/InputDialogs.tsx:392]:
```suggestion
            <Input type="select"
```
Use this to use the Goober styling

File: frontend/src/components/InputDialogs/InputDialogs.tsx [hunk 2]
```diff
@@ -389,14 +389,19 @@ const InputDialogs = () => {
...
               value={direction}
               onChange={handleDirectionIn}
-              onKeyUp={handleSave}
+            //  onKeyUp={handleSave}
               placeholder={'↔\t(direction)'}
-              style={TRANSITION_INPUT_STYLE}
-            />
+              style={{...TRANSITION_INPUT_STYLE, width: '155px'}}
+            >
+            <option value="">↔ (direction)</option>
```
Comment (SidhraFernando-Plant) [inline frontend/src/components/InputDialogs/InputDialogs.tsx:400]: <option value="">↔ (move)</option>

Comment (SidhraFernando-Plant) [inline frontend/src/components/InputDialogs/InputDialogs.tsx:400]:
```suggestion
            <option value="">↔ (move)</option>
```

File: frontend/src/components/InputDialogs/InputDialogs.tsx [hunk 3]
```diff
@@ -389,14 +389,19 @@ const InputDialogs = () => {
...
               ref={inputDirectionRef}
               value={direction}
               onChange={handleDirectionIn}
-              onKeyUp={handleSave}
+            //  onKeyUp={handleSave}
               placeholder={'↔\t(direction)'}
-              style={TRANSITION_INPUT_STYLE}
-            />
+              style={{...TRANSITION_INPUT_STYLE, width: '155px'}}
+            >
```
Comment (SidhraFernando-Plant) [inline frontend/src/components/InputDialogs/InputDialogs.tsx:399]:
```suggestion
            />
```

File: frontend/src/components/InputDialogs/InputDialogs.tsx [hunk 4]
```diff
@@ -389,14 +389,19 @@ const InputDialogs = () => {
...
+            //  onKeyUp={handleSave}
               placeholder={'↔\t(direction)'}
-              style={TRANSITION_INPUT_STYLE}
+              style={{...TRANSITION_INPUT_STYLE, width: '155px'}}
             />
+            <option value="">↔ (direction)</option>
+            <option value="L">L</option>
+            <option value="S">S</option>
+            <option value="R">R</option>
+            </select>
```
Comment (SidhraFernando-Plant) [inline frontend/src/components/InputDialogs/InputDialogs.tsx:404]:
```suggestion
```

**Pull request closed without merging**